docker/docker-compose.centos.yaml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: netty-tcnative-centos:default
+    build:
+      context: .
+      dockerfile: Dockerfile.centos
+
+  common: &common
+    image: netty-tcnative-centos:default
+    depends_on: [runtime-setup]
+    volumes:
+      - ~/.ssh:/root/.ssh
+      - ~/.gnupg:/root/.gnupg
+      - ..:/code
```

Comment:
also bind .m2? or at least settings.xml?

Reply:
@tomerd I only wanted to bind it when using `shell` as I want to refetch maven in case we install something during build.

Reply:
binding `settings.xml` sounds good tho... let me do this